util/windows/filesystem.go

```diff
@@ -0,0 +1,90 @@
+// +build windows
+
+package windows
+
+import (
+	"fmt"
+	"strings"
+	"syscall"
+	"unsafe"
+	"github.com/mackerelio/mackerel-agent/logging"
+)
+
+var windowsLogger = logging.GetLogger("windows")
+
+// CollectDfValues XXX
+func CollectDfValues() (map[string]map[string]interface{}, error) {
```

Comment:
I don't think we should name this function as `Df` as this function does not call `df`; Also we don't need to pass data by `map[string]interface{}`, currently we have type assertion inside `metrics/windows/FilesystemGenerator.Generate()`, but we might pass them by structs.